tachyon/base/numerics/README.md

```diff
@@ -370,16 +369,14 @@ all types and `%`, `<<`, `>>`, `&`, `|`, `^` for integers) along with comparison
 operators for arithmetic types of any size. However, *the [variadic template
 functions
 ](#ClampedNumeric_in-clamped_math_h-Non_member-helper-functions)
-are the prefered API,* as they remove type ambiguities and help prevent
+are the preferred API,* as they remove type ambiguities and help prevent
 a number of common errors. The variadic functions can also be more performant,
 as they eliminate redundant expressions that are unavoidable with the operator
-overloads. (Ideally the compiler should optimize those away, but better to avoid
+overloads. (Ideally, the compiler should optimize those away, but better to avoid
 them in the first place.)
 
 Type promotions are a slightly modified version of the [standard C/C++ numeric
-promotions
-](http://en.cppreference.com/w/cpp/language/implicit_conversion#Numeric_promotions)
-with the two differences being that *there is no default promotion to int*
+promotions](http://en.cppreference.com/w/cpp/language/implicit_conversion#Numeric_promotions) with the two differences being that *there is no default promotion to int*
```

Comment:
Did you to this by mistake or intend? If this was a mistake, Could you change it back to original?